test/cmake_text_tests.bzl

```diff
@@ -253,12 +253,13 @@ def _merge_flag_values_no_toolchain_file_test(ctx):
         user_env,
         [],
         cmake_commands = [],
+        cmake_prefix = "emcmake"
```

Comment:
```suggestion
        cmake_prefix = "emcmake",
```

Reply:
Oops, forgot to run buildifier :) Thanks for fixing up!